src/test/java/org/folio/notes/controller/NoteTypesControllerTest.java

```diff
@@ -39,16 +46,6 @@
 import org.springframework.web.method.annotation.MethodArgumentTypeMismatchException;
 import org.testcontainers.shaded.org.apache.commons.lang.RandomStringUtils;
 
-import org.folio.notes.client.UsersClient;
-import org.folio.notes.domain.dto.Note;
-import org.folio.notes.domain.dto.NoteType;
-import org.folio.notes.domain.dto.User;
-import org.folio.notes.domain.entity.NoteTypeEntity;
-import org.folio.notes.exception.NoteTypeNotFoundException;
-import org.folio.notes.exception.NoteTypesLimitReached;
-import org.folio.notes.support.TestApiBase;
-import org.folio.spring.cql.CqlQueryValidationException;
-
```

Comment:
Please check your IDE config. "org.folio.*" imports should be last in the import section